core/src/repair/certificate_service.rs

```diff
@@ -73,8 +72,8 @@ impl CertificateService {
     }
 
     fn receive_new_certificates(
-        certificate_receiver: &Receiver<(CertificateId, CertificateMessage)>,
-    ) -> Result<Vec<(CertificateId, CertificateMessage)>> {
+        certificate_receiver: &Receiver<(Certificate, CertificateMessage)>,
```

Comment:
Originally we did this because Certificate contains something CertificateId doesn't have. Now you use Certificate, CertificateMessage should have Certificate.

Reply:
That said, it's probably easier to just do https://github.com/anza-xyz/alpenglow/issues/299